### ISSUE TITLE
New Scripts Adding Policy

### ISSUE THREAD

**Issue by Jipok** [piqueserver/piqueserver-extras#2]
Although I got the rights to add anything without approval, I don't like the idea of uploading everything here. 
I ask everyone interested to express their opinion on: 
1) I have uploaded 6 scripts here, do you think they are suitable for this project?
2) What about armageddon.py with little code and little meaning? 
3) Should I add multiple implementations of the same script? For example, there is a hookshot script (teleportation to a visible block). Its code is very bloated and overcomplicated, I would like to rewrite it. So is it worth rewriting or posting a version that everyone uses. Or both? 
The same goes for mines and rpg(rocket).
4) Should I upload ported foreign scripts such as [AHT](http://aosjp.web.fc2.com/AHTdata/antihack.html) here?

**Comment by feikname** [comment]
1. I'm totally ok with adding new scripts, I think it's nice. 
2. Same as above
3. I'm very in favor of rewriting scripts*
4. I don't see any problem posting foreign scripts as long as credit is due and the script license allows it.
If the license doesn't allow we should link it somewhere and/or keep a nonfree folder.

*Regarding rewriting scripts, I'd suggest the following scheme: (taking platform.py as example)
1. the old `platform.py` should stay `platform.py`, with a comment on top noting this is old version and rewritten version is preferred
2. the rewritten `platform.py` is now the `platform-pique-rewritten.py`

This way there is no ambiguity possible, and stuff that reference platform.py won't break

> Although I got the rights to add anything without approval, I don't like the idea of uploading everything here.

Sending via PR is IMO preferred to imply commiting as this allows everyone to review and suggest changes.

If you were talking about polluting, an index in the README or maybe even in piqueserver docs could suffice.

**Comment by Jipok** [comment]
> Sending via PR is IMO preferred to imply commiting as this allows everyone to review and suggest changes.

It seemed to me that people have no time to do this.

**Comment by feikname** [comment]
> It seemed to me that people have no time to do this.

That is true, forgot about that. It's ok by me you commiting here whenever you see fit then.
